### PR TITLE
Robustness against time drifts

### PR DESCRIPTION
**Pull request by stefanLeo**
Using System.nanotime increases:
a) Accuracy
b) Performance
c) Is robust against time drifts

### CODE REVIEW

**Comment by sbordet** [comment]
@stefanLeo thanks! You need to comply with the [contribution guidelines](https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/CONTRIBUTING.md) for us to accept this PR.

Also, can you use `toMillis()` instead of `convert(time, MILLISECONDS)` - it's less verbose.

**Comment by joakime** [comment]
@stefan42 you will need to address the lack of Eclipse Contributor Agreement, before Eclipse Legal will allow us to accept this PR.
https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/CONTRIBUTING.md#eclipse-contributor-agreement

**Comment by sbordet** [comment]
@stefanLeo I have verified that you have a correct ECA under `stefan.leonhartsberger@gmx.net` (not `.com`).

You probably have signed other commits with a different address.

Furthermore, the latest commit contains a conflict.

So: redo your changes, make sure you sign off your commit with the proper email, and _force push_ to this branch so that previous commits are discarded and the checks run again.

File: jetty-io/src/main/java/org/eclipse/jetty/io/IdleTimeout.java [hunk 1]
```diff
@@ -61,30 +61,35 @@ public IdleTimeout(Scheduler scheduler)
...
-        return _idleTimestamp;
-    }
 
+	/**
+<<<<<<< HEAD
+	* Get the idle-for time
+=======
+	* Get the idle for time
+>>>>>>> 68c6fcf6cdb1bc617f8b0545658c8c47c77eaefe
+	*/
```
Comment (gregw) [inline jetty-io/src/main/java/org/eclipse/jetty/io/IdleTimeout.java]: This is a bad merge commit

**Comment by olamy** [comment]
@stefanLeo please do use ` git merge ` to sync from jetty-9.4.x branch this create non needed noisy commit and wrong merge. 
if you want a sync branch you must use git fetch and then git rebase.
thanks

**Comment by joakime** [comment]
Closing, using PR#3535 instead.

**Pull request closed without merging**